docs/trust.md

```diff
@@ -1,11 +1,11 @@
 AEA applications have different requirements for _trustlessness_ or _trust minimisation_.
 
-For example, using the AEA <a href="../weather-skills/">weather skills demo</a> without ledger payments means that clients must trust that any data the weather station sends is sufficient, including no data at all. Similarly, the weather station must trust the weather clients to send payment via some mechanism.
+For example, using the AEA <a href="../weather-skills/">weather skills demo</a> _without_ ledger payments means that the client has no way of validating the data it receives from the weather station (including no data at all) and so must accept anything the weather station sends. Similarly, the station must trust that the client somehow sends the payment amount that they agree.
```

Comment:
There are two issues here: that something is sent (data/payment) and that what is sent is the agreed upon.